network-store-model/src/main/java/com/powsybl/network/store/model/LimitSelector.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Charly Boutier <charly.boutier at rte-france.com>
+ */
+public interface LimitSelector {
+
+    default LimitsAttributes getLimits(TemporaryLimitType type, int side) {
+        if (!getSideList().contains(side)) {
```

Comment:
Maybe this check is not necessary (and could slow down access). I would have handle this exception in getCurrentLimits, getApparentPowerLimits and getActivePowerLimits

Reply:
Ok done